Gemfile

```diff
@@ -31,12 +32,17 @@ group :development do
   gem 'web-console'
 end
 
+# rubocop:disable Layout/LineLength
 gem 'qonsole-rails', git: 'https://github.com/epimorphics/qonsole-rails'
-# gem 'qonsole-rails', path: '/home/ian/projects/epimorphics/qonsole-rails'
+# gem 'qonsole-rails', path: '~/Epimorphics/clients/land-registry/projects/qonsole-rails'
+
+# TODO: For running the app locally for testing you can set this to your local path
+# gem 'json_rails_logger', '~> 0.3.5', path: '~/Epimorphics/shared/json-rails-logger/'
+# gem 'lr_common_styles', '~> 1.9.1', path: '~/Epimorphics/clients/land-registry/projects/lr_common_styles/'
+# rubocop:enable Layout/LineLength
 
+# TODO: In production you want to set this to the gem from the epimorphics package repo
```

Comment:
Is it possible to divide a Gemfile up by environment? @jonrandahl

Reply:
@joescottdave It is, and I did think of this, however usually as we're only changing to the "local" version of the gems while we're developing the gems and then testing in the dependent apps I felt this was the more appropriate approach.